Fix build failure with GCC 10 due to duplicate definitions.  It can be
reproduced on older compilers with -fno-common.

Fix taken from Gentoo: https://bugs.gentoo.org/705898

--- a/src/parseopt.c
+++ b/src/parseopt.c
@@ -255,8 +255,6 @@ print_option_descr (const char *descr, size_t lmargin, size_t rmargin)
 }
 
 char *parseopt_program_name;
-char *parseopt_program_doc;
-char *parseopt_program_args;
 const char *program_bug_address = "<" PACKAGE_BUGREPORT ">";
 void (*parseopt_help_hook) (FILE *stream);
 
